packages/world/mud.config.mts

```diff
@@ -80,8 +80,8 @@ const config: StoreUserConfig = {
     },
     InstalledModules: {
       primaryKeys: {
-        namespace: SchemaType.BYTES16,
-        mdouleName: SchemaType.BYTES16,
+        moduleName: SchemaType.BYTES16,
+        configHash: SchemaType.BYTES32, // Hash of the params passed to the `install` function
```

Comment:
nit: what about `argsHash` to align with  other variables? "config" seems a little overloaded here